src/Tribe/Support.php

```diff
@@ -164,15 +164,17 @@ public function getSupportStats() {
 				$php_info[ $php_var ] = $val;
 			}
 
+			$site_url = get_site_url();
 			$systeminfo = array(
 				'Home URL'               => get_home_url(),
-				'Site URL'               => get_site_url(),
+				'Site URL'               => $site_url,
 				'Site Language'          => get_option( 'WPLANG' ) ? get_option( 'WPLANG' ) : esc_html__( 'English', 'tribe-common' ),
 				'Character Set'          => get_option( 'blog_charset' ),
 				'Name'                   => $user->display_name,
 				'Email'                  => $user->user_email,
 				'Install keys'           => $keys,
 				'WordPress version'      => get_bloginfo( 'version' ),
+				'Permalink Structure'    => $site_url . get_option( 'permalink_structure' ),
```

Comment:
Why send the Permalink wit the Site URL? Instead of just the DB option alone?

Reply:
To make it look like the one on the Admin panel.

- https://cl.ly/2S192K3u0z3G

Nothing in particular really, but let me know if the option alone is best.